packages/js/data/changelog/add-33456_shipping_class_data_store_required_fields

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Update CrudActions type to allow for required fields. #33809
```

Comment:
Not sure if we need a changelog or not since this is all in the same release, right?

Reply:
True, I could remove the changelog